app/models/inzending-voor-toezicht.js

```diff
@@ -3,40 +3,40 @@ import attr from 'ember-data/attr';
 import { belongsTo, hasMany } from 'ember-data/relationships';
 
 export default Model.extend({
-  created: attr('datetime'),
-  modified: attr('datetime'),
-  sentDate: attr('datetime'),
-  description: attr(),
-  remark: attr(),
-  temporalCoverage: attr(),
+  accountAcceptanceStatus: belongsTo('toezicht-account-acceptance-status', { inverse: null }),
```

Comment:
It's OK to sort them alphabetically (but then we should maybe introduce this convention in all our models) but I would keep the attributes, relationships and computed properties (if any) in separate blocks.